src/lib/Components/Consignments/Screens/Overview.tsx

```diff
@@ -45,6 +45,8 @@ interface State extends ConsignmentSetup {
   hasLoaded?: boolean
 }
 
+const track: Track<Props, State> = _track
+@track()
```

Comment:
you don't want context screen here? You've defined the `ConsignmentsOverview` page view const :o.